controllers/user.js

```diff
@@ -100,31 +98,44 @@ exports.EditUsername = async (req, res) => {
     });
   }
 
-  User.findOne({ _id: req.body._id }, (err, user) => {
+  const regex = RegExp('^(?=[a-zA-Zก-๛_\d]*[a-zA-Zก-๛])[-a-zA-Zก-๛0-9_\d]{5,12}$');
+  const usernameValidate = regex.test(body.username);
+
+  if (body.username == null || body.username == "" || body.username == " ") {
```

Comment:
คิดว่าตรงนี้ใช้ username.trim() เอาดีกว่านะ แล้วค่อยเช็คว่า length เป็น 0 รึเปล่า
แบบว่า if (username == null หรือ username.trim().length == 0 ) ไรงี้
https://medium.com/@ugwuraphael/javascript-properties-and-methods-string-prototype-trim-7806d765188

Reply:
แก้แล้วนะค้าบ